src/count.js

```diff
@@ -20,6 +20,9 @@ function count(ast) {
     case 'keyframes':
     case 'import':
     case 'supports':
+    case 'charset':
+    case 'namespace':
+    case 'page':
```

Comment:
The problem with your `over-limit-with-at-rules` test is that the 4096th selector is not an `@` selector. The selector you are using in your test would already split the css to a separate file. To truly test this you should do a test for each `@` selector as the 4096th selector then ensure all is well in the versions of IE you mentioned. The purpose of this function is to determine if the current selector will push the selector count above the 4095 limit imposed in IE. If you are returning `0` for the selector count you are saying that if `@page` selector does not count against that limit of 4095 characters, which I'm not entirely confident is correct.